fixtures/webstudio-remix-netlify-edge-functions/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "netlify serve",
+    "typecheck": "tsc -b",
+    "predev": "rimraf ./public/_redirects"
+  },
+  "dependencies": {
+    "@remix-run/react": "^1.19.2",
+    "@remix-run/server-runtime": "^1.19.2",
+    "@remix-run/node": "^1.19.2",
+    "@webstudio-is/react-sdk": "workspace:*",
+    "@webstudio-is/sdk-components-react-radix": "workspace:*",
+    "@webstudio-is/sdk-components-react-remix": "workspace:*",
+    "@webstudio-is/sdk-components-react": "workspace:*",
+    "@webstudio-is/form-handlers": "workspace:*",
+    "@webstudio-is/image": "workspace:*",
+    "@webstudio-is/sdk": "workspace:*",
+    "isbot": "^3.6.8",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "@netlify/functions": "^1.3.0",
+    "@netlify/edge-functions": "^2.0.0",
+    "@netlify/remix-edge-adapter": "1.2.0",
+    "cross-env": "^7.0.3"
+  },
+  "devDependencies": {
+    "@remix-run/serve": "^1.19.2",
+    "@remix-run/dev": "^1.19.2",
+    "@types/react": "^18.2.20",
+    "@types/react-dom": "^18.2.7",
+    "@webstudio-is/http-client": "workspace:*",
+    "eslint": "^8.48.0",
+    "typescript": "5.2.2",
+    "rimraf": "^4.1.4",
```

Comment:
Node: supports fs.rm with recursive flag
rimraf:
<img width="845" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5635476/c19c3307-47cb-4466-91ef-06edb9f2c1c8">

Reply:
I get it, but don't we need 1:1 with the original template from netlify. Or we just want to deploy to netlify and make modifications as we need ?

Reply:
Wow, we can do this now
```
node -e "fs.rmSync('./docs', { recursive: true, force: true })"
```

Reply:
Still here